packages/common-all/src/utils/index.ts

```diff
@@ -120,6 +120,14 @@ export function isFalsy(u: any): boolean {
   return _.some([_.isUndefined(u), _.isEmpty(u), _.isNull(u)]);
 }
 
+/**
+ * `do` expession
+ * [ECMAScript proposal: `do` expressions](https://github.com/tc39/proposal-do-expressions)
+ */
+export function do_<T>(f: () => T): T {
```

Comment:
this goes back to - lets talk before introducing new paradigms. for this, would not do this but use a named function. has the benefit of being easier to debug

Reply:
[`df89346` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/df89346508c9a97a858507b33df7712197a73129)